django/contrib/sessions/backends/db.py

```diff
@@ -95,6 +140,39 @@ def save(self, must_create=False):
                 raise UpdateError
             raise
 
+    async def asave(self, must_create=False):
+        """
+        Save the current session data to the database. If 'must_create' is
+        True, raise a database error if the saving operation doesn't create a
+        new entry (as opposed to possibly updating an existing entry).
+        """
+        if self.session_key is None:
+            return await self.acreate()
+        data = await self._aget_session(no_load=must_create)
+        obj = await self.acreate_model_instance(data)
+        using = router.db_for_write(self.model, instance=obj)
+        try:
+            # This code MOST run in a transaction, so it requires sync_to_async
+            # until `transaction.atomic` supports async
+            @sync_to_async
+            def sync_transaction():
+                with transaction.atomic(using=using):
+                    obj.save(
+                        force_insert=must_create,
+                        force_update=not must_create,
+                        using=using,
+                    )
+
+            await sync_transaction()
```

Comment:
this may need to be written with `thread_sensitive=True` with the `sync_to_async`

Reply:
I think it defaults to `True` now, but that can't hurt